lib/physicians/validation.ts

```diff
@@ -81,14 +81,21 @@ export const requestPhysicianInformationSchema = object({
   mspNumber: string()
     .matches(/^\d+$/, 'Must only contain numbers')
     .required('Please enter the MSP number'),
-  phone: string().min(10, 'Must be a valid phone number').required('Please enter a phone number'),
+  phone: string()
+    .required('Please enter a phone number')
+    .matches(
+      /(^(\(?\d{3}\)?-?\d{3}-?\d{4}$))/,
```

Comment:
I notice we use the same regex for phone number and postal code for everywhere. Let's put those regex patterns in a file `lib/validation/utils.ts` and import from there. That way, if any of the regexes need to be changed in the future, it's an easy fix.